src/twisted/spread/test/test_pb.py

```diff
@@ -170,6 +169,52 @@ def disconnectClientFactory():
 
 
 
+@implementer(interfaces.IConnector)
+class ReconnectingConnector(object):
```

Comment:
Please file a ticket for putting this into `proto_helpers`.  Given that this is a fake of a core Twisted interface, it shouldn't be squirreled away inside PB's tests.